group_vars/all

```diff
@@ -111,6 +111,13 @@
   easybuild_buildpath: "/tmp/build"
   easybuild_sourcepath: "/tmp/source"
 
+#matlab download
+  matlab_provision: false
+  matlab_download_url: "https://uab.box.com/shared/static/y01qu7oo1gpne6j2s6nqwcuee63epivo.gz"
+  matlab_clustershare: "/opt/ohpc/pub/apps/matlab/"
+  matlab_destination: "/tmp/matlab.tar.gz"
+  matlab_module_dir: "/export/eb/modules/all/matlab" 
```

Comment:
I would define a variable for module name since we use different module names on dev and cheaha.